### PR TITLE
allow users to start up with degraded cassandra clusters

### PR DESCRIPTION
**Pull request by clockfort**

### CODE REVIEW

**Comment by tmgordeeva** [comment]
would be nice if the unresponsive ones were autoblacklisted.

**Comment by tmgordeeva** [comment]
should also do the same health checks when a new node entries via auto refresh

**Comment by clockfort** [comment]
agreed. done and done.

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java [hunk 1]
```diff
@@ -207,6 +207,7 @@ private boolean isHostHealthy(InetSocketAddress host) {
         try {
             CassandraClientPoolingContainer testingContainer = new CassandraClientPoolingContainer(host, config);
             testingContainer.runWithPooledResource(describeRing);
+            testingContainer.runWithPooledResource(CassandraVerifier.healthCheck);
```
Comment (tmgordeeva) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java]: also validatePartitioner if this succeeds?

Comment (clockfort) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java]: oh whoops, this whole thing is just supposed to be validate partitioner
dammit

**Pull request merged**